libsql-client/src/connection.rs

```diff
@@ -89,17 +89,15 @@ impl Connection for GenericConnection {
 /// Establishes a database connection based on environment variables
 ///
 /// # Env
-/// * `LIBSQL_CLIENT_BACKEND` - one of the available backends; e.g. `reqwest`, `local`, `workers`
 /// * `LIBSQL_CLIENT_URL` - URL of the database endpoint - e.g. a https:// endpoint for remote connections (with specified credentials) or local file:/// path for a local database
+/// * (optional) `LIBSQL_CLIENT_BACKEND` - one of the available backends; e.g. `reqwest`, `local`, `workers`
```

Comment:
Is there a default value that we could mention here?

Reply:
yup, I'll mention in the comment that the driver tries to guess the backend based on the URL